test_runner/batch_others/test_tenant_relocation.py

```diff
@@ -0,0 +1,174 @@
+from contextlib import closing, contextmanager
+import os
+import pathlib
+import subprocess
+from fixtures.log_helper import log
+import time
+import requests
+import signal
+
+from fixtures.zenith_fixtures import PgProtocol, PortDistributor, ZenithEnvBuilder, zenith_binpath, pg_distrib_dir
+
+
+@contextmanager
+def new_pageserver_helper(new_pageserver_dir: pathlib.Path,
+                          pageserver_bin: pathlib.Path,
+                          remote_storage_mock_path: pathlib.Path,
+                          pg_port: int,
+                          http_port: int):
+    """
+    cannot use ZenithPageserver yet bacause it depends on zenith cli
+    which currently lacks support for multiple pageservers
+    """
+    cmd = [
+        str(pageserver_bin),
+        '--init',
+        '--listen-pg',
+        f'localhost:{pg_port}',
+        '--listen-http',
+        f'localhost:{http_port}',
+        '--workdir',
+        str(new_pageserver_dir),
+        '--postgres-distrib',
+        pg_distrib_dir,
+    ]
+
+    subprocess.check_output(cmd, text=True)
+
+    new_pageserver_config_file_path = new_pageserver_dir / 'pageserver.toml'
+
+    # add remote storage mock to pageserver config
```

Comment:
Maybe worth a todo, IMO, we would like to enable remote storage for every Python IT tests with one env var/command, so the file appending way is rather crude and temporary.